File: poem-scraper.js
const request = require('request');
const cheerio = require('cheerio');
let url = "https://poemasinfantis.blogs.sapo.pt";

request(url, (error, response, body) => {
    const $ = cheerio.load(body);
    const limit = 10;
    const poems = $("#posts").children().slice(2, limit).toArray();
    for(let poem of poems) {
        const textLines = $(".posttext", poem).text().trim().split("\n");
        const title = textLines.shift();
        const text = textLines.join("\n").trim();
        console.log(title);
        console.log();
        console.log(text);
});

File: poem-scraper.py
import requests_html


session = requests_html.HTMLSession()
response = session.get("https://poemasinfantis.blogs.sapo.pt")
poems = response.html.find("#posts .posttext")
for poem in poems[2:5]:
    lines = poem.text.split("\n")
    title = lines.pop(0)
    text = "\n".join(lines[2:]).strip()
    print(title)
    print()
    print(text)
    print("=====")
